src/authentication/authentication-hapi-plugin.ts

```diff
@@ -267,3 +303,9 @@ export function generatePassword(length = 16) {
 export function sanitizeUsername(username: string) {
   return username.replace('|', '-');
 }
+
+export function assertTeam(teamNameOrId: string |  number, team: Group) {
+  return team.name.toLowerCase() === teamNameOrId.toString()
+    || team.path === teamNameOrId.toString()
```

Comment:
It's a "pathified" presentation of the team name, all lowercase and spaces converted to dashes.